### ISSUE TITLE
v0.17.0 upgrade fails (ES Module / stylistic)

### ISSUE THREAD

**Issue by Nikerabbit** [wikimedia/stylelint-config-wikimedia#233]
> Warning: Running stylelint failed
Error [ERR_REQUIRE_ESM]: require() of ES Module /home/runner/work/mediawiki-extensions-Nimiarkisto/mediawiki-extensions-Nimiarkisto/node_modules/@stylistic/stylelint-plugin/lib/index.js from /home/runner/work/mediawiki-extensions-Nimiarkisto/mediawiki-extensions-Nimiarkisto/node_modules/stylelint/lib/augmentConfig.js not supported.
Instead change the require of index.js in /home/runner/work/mediawiki-extensions-Nimiarkisto/mediawiki-extensions-Nimiarkisto/node_modules/stylelint/lib/augmentConfig.js to a dynamic import() which is available in all CommonJS modules.

**Comment by supertassu** [comment]
This sounds like you're upgrading `stylelint-config-wikimedia` but not `grunt-stylelint` at the same time.

**Comment by Nikerabbit** [comment]
Is this documented somewhere? I saw nothing in release notes and both upgrades (as the bot is not combining updates) are failing separately, so it's not very intuitive.

**Comment by jdforrester** [comment]
> Is this documented somewhere? I saw nothing in release notes and both upgrades (as the bot is not combining updates) are failing separately, so it's not very intuitive.

When you try to upgrade one without the other, npm will alert you with incompatible dependencies. The same as happened for #231. Is your npm not giving you these warnings?

**Comment by Nikerabbit** [comment]
No, I am not seeing any warnings about incompatible dependencies.

**Comment by Volker-E** [comment]
@Nikerabbit Please try again with upcoming stylelint-config-wikimedia release.

**Comment by Volker-E** [comment]
Looks like job done with https://github.com/Nikerabbit/mediawiki-extensions-Termbank/commit/85a825baec073e59698414e9a2f0b341ee445e7a and https://github.com/Nikerabbit/mediawiki-extensions-Nimiarkisto/commit/bceb5a6327db8c4d4128b3f8b572bf17844bc520
